bool save_to_file
---
geometry_msgs/PoseStamped print_origin